File: ts/1792/1678.ts
function interpret(command: string): string {
  const len = command.length
  let str = '', i = 0
  while (i < len) {
    if (command[i] === '(') {
      if (command[i + 1] === 'a') {
        str += 'al'
        i += 3
      } else {
        str += 'o'
        i += 1
      }
    }
    else str += command[i]
    i += 1
  }
  return str
};

File: py3/1792/1678.py
class Solution:
    def interpret(self, command: str) -> str:
        ans, i = '', 0
        while i < len(command):
            if command[i:i + 2] == '()':
                ans += 'o'
                i += 2
            elif command[i:i + 4] == '(al)':
                ans += 'al'
                i += 4
            else:
                ans += command[i]
                i += 1
        return ans